test/model/message_test.dart

```diff
@@ -576,4 +584,346 @@ void main() {
         .reactions.isNotNull().jsonEquals([eg.unicodeEmojiReaction]);
     });
   });
+
+  group('handleSubmessageEvent', () {
```

Comment:
Oh, one other nit:

This group isn't specific to `handleSubmessageEvent` — it's about interpreting Message objects that come in the first place with submessages, too, where `SubmessageEvent` may never be involved.

Reply:
Gave the outer group a different name: "handle Poll related events". The name `handleSubmessageEvent` should be applicable to the subgroup of tests that do take care of `SubmessageEvent`. To parallel that, I renamed the other subgroup to "handleMessageEvent with initial submessages"